src/app/index.spec.tsx

```diff
@@ -45,6 +45,32 @@ describe('create app', () => {
     expect(app.history.location.state).toEqual('asdf');
   });
 
+  describe('inside the browser', () => {
+    const windowBackup = window;
+    jest.unmock('../helpers/Sentry');
+
+    beforeEach(() => {
+      (global as any).window = windowBackup;
+    });
+
+    afterEach(() => {
+      delete (global as any).window;
+    });
```

Comment:
What does this accomplish? The window is there in the first place, right? Why does it need to be cleaned after each test?

Reply:
i think this was copied from below where it does the reverse, deleting the window for each test and then restoring it. i don't think its necessary here